setup.py

```diff
@@ -190,6 +191,11 @@ def run(self):
             elif os.path.isdir(f):
                 shutil.rmtree(f)
 
+
+        for name in glob.glob(dir_setup+"/doc/src/modules/physics/vector/*.pdf"):
+            if os.path.isfile(name):
+                os.remove(name)
```

Comment:
I had a problem with `glob` on windows an year ago so I had to use `os.listdir` and `os.walk`. I don't remember the cause and the reference that I used. Sorry for blind suggestion.